packages/client/src/api.ts

```diff
@@ -63,6 +112,15 @@ export abstract class MessagingChannelApi extends MessagingChannelBase {
     ).data.map((x) => this.deserializeConversation(x))
   }
 
+  /**
+   * Sends a message to the messaging server
+   * @param clientId id of the client that owns the conversation
+   * @param conversationId id of the conversation to post the message to
+   * @param authorId id of the message autor. `undefined` if bot
+   * @param payload content of the message
+   * @param flags message flags
```

Comment:
What are those?

Reply:
For converse

Reply:
Can you add a comment that adds this detail!